edge/services/volantmq/horizon/service.definition.json

```diff
@@ -0,0 +1,33 @@
+{
+	"org": "${HZN_ORG_ID}",
+	"label": "volantmq",
+	"description": "VolantMQ MQTT server",
+	"documentation": "https://volantmq.io",
+	"url": "ibm.volantmq",
+	"version": "${SERVICE_VERSION}",
+	"arch": "${ARCH}", 
+	"public": false,
+	"sharable": "singleton",
+	"requiredServices": [],
+	"userInput": 
+	[
+	{ 
+		"name": "VOLANTMQ_USERS", 
+		"label": "", 
+		"type": "string", 
+		"defaultValue": "fft-server: 7b1bf1e4f9535de960093f1c303fe35f49167bdc103ba99ad7dc9d62e2807a1d\nfft-client: fbfc2da74af1af1945ba7bf403cde789091e39b13c420170080872323dd2d148" 
+	}
+	],
+	"deployment": {
+		"services": {
+			"volantmq": {
+				"image": "${DOCKER_IMAGE_BASE}:${ARCH}-${SERVICE_VERSION}",
+				"privileged": false,
+				"specific_ports": [{
```

Comment:
I'm not sure if there's an actual issue here but I you look in the documentation for deployment strings the values here should be `ports` https://github.com/open-horizon/anax/blob/master/doc/deployment_string.md